### PR TITLE
Change Pico Modes settings

### PR DESCRIPTION
**Pull request by MustafaMustafa**
This pull request will include changes to enable setting pico mode from BFC chain. 

I think we should have two IO bfc chain options: PicoWrite, PicoRead. Depending on the option we can use StMaker::SetMode() in the StBfChain to set it to read or write. 

I will work on the Vtx mode now. I will try to use SetAttr. 

P.S. I really wanted to make the IoMode a strong enum, but rootcint ...

### CODE REVIEW

File: StPicoDstMaker/StPicoDstMaker.cxx [hunk 1]
```diff
@@ -206,6 +217,20 @@ Int_t StPicoDstMaker::Init()
   return kStOK;
 }
 
+int StPicoDstMaker::setVtxModeAttr()
+{
+  if(TObject* vtxMode = (TObject*)(GetAttr()->FindObject("picovtxmode")))
```
Comment (plexoos) [inline StPicoDstMaker/StPicoDstMaker.cxx]: StMaker::SAttr() ?

Comment (MustafaMustafa) [reply StPicoDstMaker/StPicoDstMaker.cxx]: SAttr is a very informative name for a method ;-)
I didn't notice it. I will use it.

**Comment by MustafaMustafa** [comment]
@plexoos 

With these changes we need two add the following bfc chain options:

```
PicoWrite ---> SetMode(1);
PicoRead ---> SetMode(2);

if (GetOption("PicoVtxAuAu200"))
{
      maker->SetAttr("picoVtxMode", "PicoVtxAuAu200");
}

if(GetOption("PicoVtxDefault"))
{
       maker->SetAttr("picoVtxMode", "PicoVtxDefault");
}
```

**Comment by plexoos** [comment]
Yes. AFAIK Jerome volunteered to do that

**Comment by MustafaMustafa** [comment]
@plexoos, I will be away for a few hours. If you approve please merge.

File: StPicoDstMaker/StPicoDstMaker.cxx (hunk 1 above)
Comment (MustafaMustafa) [reply StPicoDstMaker/StPicoDstMaker.cxx]: Done in     d27ba1a.

**Comment by MustafaMustafa** [comment]
@plexoos,

I take it that you don't have further comments on this pull-request?

I can merge and commit to CVS.

**Comment by plexoos** [comment]
Sorry, just got the note... looks good to me, please go ahead and merge

**Pull request merged**